### ISSUE TITLE
problems encountered on my own dataset

### ISSUE THREAD

**Issue by puppy2000** [282857341/nnFormer#97]
hello,I want to ask where should I modify if I want to train nnFormer on my own dataset?
![image](https://user-images.githubusercontent.com/55677296/198950465-3865fb22-32a3-40ac-ac46-b9a683d60646.png)
In the nnFormerTrainerV2_nnformer_acdc.py,the do_split() function,what are these code intended for?
![image](https://user-images.githubusercontent.com/55677296/198950784-729a5506-23fd-43e4-9ed8-f5b44cc9d4bf.png)

**Comment by 282857341** [comment]
This part of code is used for dividing the training set and validation set.
